core-api/src/test/java/com/optimizely/ab/odp/ODPEventManagerTest.java

```diff
@@ -255,6 +255,157 @@ public void validateEventData() {
         assertFalse(event.isDataValid());
     }
 
+    @Test
+    public void validateAugmentCommonData() {
```

Comment:
update the header of this file.

Reply:
We have to find a way to automate these header updates :)